crates/rune/src/build.rs

```diff
@@ -74,10 +74,59 @@ pub struct Build<'a> {
     context: Option<&'a Context>,
     diagnostics: Option<&'a mut Diagnostics>,
     options: Option<&'a Options>,
-    visitor: Option<&'a mut dyn compile::CompileVisitor>,
+    visitors: Vec<&'a mut dyn compile::CompileVisitor>,
     source_loader: Option<&'a mut dyn SourceLoader>,
 }
 
+/// Wraps a collection of CompileVisitor
+struct CompileVisitorGroup<'a> {
```

Comment:
This was required to avoid a `&[& ...]` situation in every API currently taking CompileVisitor.